setting-up-ocarina.md

```diff
@@ -0,0 +1,120 @@
+
+### 0. Register for an account via Majora
+[Read the instructions for how to register](register)
+
+### 1. Install the Ocarina client
+
+[Ocarina](https://github.com/SamStudio8/ocarina/tree/master/ocarina) is a command line tool that is used to connect to Majora and perform actions with elevated privileges that are not possible on the website.
+It is not terribly difficult to use, but ideally you will have used a command line tool before.
+
+You can install the latest version with the Python package manager. You'll probably want to install it into a conda environment on the shared node.
+
+```
+conda create -n ocarina python=3.7
+conda activate ocarina
+pip install git+https://github.com/samstudio8/ocarina.git
+```
+
+Please remember to check the [ocarina changelog](https://github.com/SamStudio8/ocarina/blob/master/CHANGELOG.md) periodically and update your client as necessary.
+
+### 2. Register your Ocarina instance as an OAuth application
+
+Instructions to do this are located [here](oauth-app). Only the first section is necessary to set up ocarina but must be done before moving to the next step.
+
+### 3a. Set your Ocarina credentials with OAuth (Recommended)
+
+Create a JSON file in your home directory named `.ocarina`. Note the starting dot.
+
+```
+{"MAJORA_DOMAIN": "https://majora.covid19.climb.ac.uk/", "MAJORA_USER": "your-username", "MAJORA_TOKEN": "OAUTH", "CLIENT_ID": "your-client-id", "CLIENT_SECRET": "your-client-secret"}
+```
+
+Where:
+* `MAJORA_DOMAIN` points to either the domain of the real, or test Majora. Do not miss the `/` at the end of the URL.
+* `MAJORA_USER` is your username on the appropriate Majora instance
+* `MAJORA_TOKEN` must be set to `OAUTH`
+* `CLIENT_ID` is the ocarina Client ID registered with majora [accessible at this majora page](https://majora.covid19.climb.ac.uk/o/applications/)
+* `CLIENT_SECRET` is the ocarina Secret ID registered with majora [accessible at this majora page](https://majora.covid19.climb.ac.uk/o/applications/)
+
+OAuth has been the standard way to authenticate since September 2020 but legacy API keys are still supported, [instructions here.](getting-api-keys)
+
+### 3b. Set your Ocarina credentials with a legacy API key
+
+[Generate a legacy API key](getting-api-keys) if you haven't already.
+
+Create a JSON file in your home directory named `.ocarina`. Note the starting dot.
+
+```
+{"MAJORA_DOMAIN": "https://majora.covid19.climb.ac.uk/", "MAJORA_USER": "your-username", "MAJORA_TOKEN": "YOUR_API_KEY", "CLIENT_ID": "", "CLIENT_SECRET": ""}
+```
+
+Where:
+* `MAJORA_DOMAIN` points to either the domain of the real, or test Majora. Do not miss the `/` at the end of the URL.
+* `MAJORA_USER` is your username on the appropriate Majora instance
+* `MAJORA_TOKEN` taken from [this majora page](https://majora.covid19.climb.ac.uk/keys/list/)
+
+OAuth has been the standard way to authenticate since September 2020 but legacy API keys are still supported, [instructions here.](getting-api-keys)
```

Comment:
can you add a reminder that API keys expire every few days and need to be renewed by following the legacy API docs (then link to them again), perhaps bold this

Reply:
pls also add "Some features (such as reading secure dataviews) are only available via OAuth authentication and will not work with the legacy API keys."